src/topology/uniform_space/cauchy.lean

```diff
@@ -277,6 +285,32 @@ begin
   exact hN (hf hmn.1 hmn.2)
 end
 
+lemma is_complete_iff_cluster_pt {s : set α} :
+  is_complete s ↔ ∀ l, cauchy l → l ≤ 𝓟 s → ∃ x ∈ s, cluster_pt x l :=
+forall₃_congr $ λ l hl hls, exists₂_congr $ λ x hx, le_nhds_iff_adhp_of_cauchy hl
+
+lemma is_complete_iff_ultrafilter {s : set α} :
+  is_complete s ↔ ∀ l : ultrafilter α, cauchy (l : filter α) → ↑l ≤ 𝓟 s → ∃ x ∈ s, ↑l ≤ 𝓝 x :=
```

Comment:
I prefer type ascription instead of mysterious up arrows, but here I admit it would be a bit painful.